From foo@baz Fri Apr 17 09:50:08 PDT 2009
Date: Fri, 17 Apr 2009 09:50:08 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: comedi: fix build errors in pcmcia comedi drivers

This fixes a few minor build errors that were previously undetected in
the comedi pcmcia drivers.

Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/ni_daq_dio24.c |    2 +-
 drivers/staging/comedi/drivers/ni_labpc_cs.c  |    2 +-
 drivers/staging/comedi/drivers/ni_mio_cs.c    |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/ni_daq_dio24.c
+++ b/drivers/staging/comedi/drivers/ni_daq_dio24.c
@@ -37,7 +37,7 @@ This is just a wrapper around the 8255.o
 the PCMCIA interface.
 */
 
-/* #define LABPC_DEBUG   /*  enable debugging messages */ */
+/* #define LABPC_DEBUG */   /*  enable debugging messages */
 #undef LABPC_DEBUG
 
 #include "../comedidev.h"
--- a/drivers/staging/comedi/drivers/ni_labpc_cs.c
+++ b/drivers/staging/comedi/drivers/ni_labpc_cs.c
@@ -60,7 +60,7 @@ NI manuals:
 */
 
 #undef LABPC_DEBUG
-/* #define LABPC_DEBUG   /*  enable debugging messages */ */
+/* #define LABPC_DEBUG */   /*  enable debugging messages */
 
 #include "../comedidev.h"
 
--- a/drivers/staging/comedi/drivers/ni_mio_cs.c
+++ b/drivers/staging/comedi/drivers/ni_mio_cs.c
@@ -66,7 +66,7 @@ See the notes in the ni_atmio.o driver.
 
 #define MAX_N_CALDACS 32
 
-static const ni_board ni_boards[] = {
+static const struct ni_board_struct ni_boards[] = {
       {device_id:0x010d,
 	      name:	"DAQCard-ai-16xe-50",
 	      n_adchan:16,
